apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -31,9 +31,6 @@ export default function RubricContainer({
   const canProvideFeedback = !!studentLevelInfo && onLevelForEvaluation;
 
   const [selectedTab, setSelectedTab] = useState(TAB_NAMES.RUBRIC);
-  const [teacherHasEnabledAi, setTeacherHasEnabledAi] = useState(
```

Comment:
When I originally created `RubricSettings`, I had added a toggle to run AI eval on and off. I removed that toggle but didn't remove this unnecessary state. Let me know if I should break this out into a separate PR.

Reply:
this seems fine to include given the size of the PR. thanks for catching this